biz.aQute.tester.junit-platform/bnd.bnd

```diff
@@ -20,7 +20,7 @@ Import-Package: \
 	aQute.*;resolution:=optional,\
 	junit.*;version="${range;[==,5);${junit3.version}}";resolution:=optional,\
 	org.apache.felix.service.command;resolution:=optional,\
-	org.junit.platform.*;version="[${junit.platform.tester.version},1.13)",\
+	org.junit.platform.*;version="[${junit.platform.tester.version},1.13)";resolution:=optional,\
```

Comment:
Isn't a hard coded `1.13` just waiting to bite you again in the future?  I mean at some point one expects the version to change and it might be that `org.junit.platform.tester.version >= 1.13` and then this will be completely bogus/broken and that might go unnoticed...

Maybe it should be more like one of these building a range around the current version:
```
${range;[==,+);${junit.platform.tester.version}}
${range;[==,=+);${junit.platform.tester.version}}
```

Have you really verified/tested that 1.14.1 does not work and needed to be excluded?  Excluded something within the major version bounds seems generally questionable.

Reply:
> I'm concerned that any upper bound other than 2 on any non-optional junit dependency will cause problems for the platform in the future. E.g., if 1.14.2 comes out and the Platform cannot update because BND restricts that, it will be probably and we might not noticed until later.

In #7000 the range is < 2 again

```
org.junit.platform.*;version="${range;[==,+);${junit.platform.tester.version}}",\
```